### PR TITLE
tooooo looong

### PR DESCRIPTION
**Pull request by Mazdallier**
Why don't you use this like forge ?
https://crowdin.com/project/minecraft-forge

### CODE REVIEW

**Comment by Thog** [comment]
1) Crowdin is not free.
2) Squash your commits.

File: src/main/resources/assets/botania/lang/fr_FR.lang [hunk 1]
```diff
@@ -121,34 +140,35 @@ botania.rank5=&6SS
...
-botania.armorset.will3.desc=Critical Hits apply a strong slowness effect (Will of Torag)
-botania.armorset.will4.desc=Critical Hits pierce through armor (Will of Verac)
-botania.armorset.will5.desc=Critical Hits apply a wither effect (Will of Karil)
-botania.armorset.will0.shortDesc=Critical Hits apply a strong weakness effect
-botania.armorset.will1.shortDesc=Critical Hit damage increases with lower HP
-botania.armorset.will2.shortDesc=Critical Hits heal for a portion of damage dealt
-botania.armorset.will3.shortDesc=Critical Hits apply a strong slowness effect
-botania.armorset.will4.shortDesc=Critical Hits pierce through armor
-botania.armorset.will5.shortDesc=Critical Hits apply a wither effect
+botania.armorset.terrasteel.name=Savoir des Ancients
```
Comment (Thog) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: It's not 'Ancient' but 'Ancients'

File: src/main/resources/assets/botania/lang/fr_FR.lang [hunk 2]
```diff
@@ -121,34 +140,35 @@ botania.rank5=&6SS
...
+botania.armorset.terrasteel.desc0=Coût en Mana diminué de 20% sur les outils à Mana
+botania.armorset.terrasteel.desc1=Régène même si la faim n'est pas remplie
+botania.armorset.will0.desc=Les coups critiques appliquent un effet de faiblesse fort (Volonté d'Ahrim)
+botania.armorset.will1.desc=Les coups critiques augmentent avec un HP bas (Volonté de Dharok)
+botania.armorset.will2.desc=Les coups critiques guérit pour une partie des dégâts infligés (Volonté de Guthan)
+botania.armorset.will3.desc=Les coups critiques appliquent un effet de lenteur fort (Volonté de Torag)
+botania.armorset.will4.desc=Les coups critiques transperce l'armure (Volonté de Verac)
+botania.armorset.will5.desc=Les coups critiques appliquent un effet de wither (Volonté de Karil)
+botania.armorset.will0.shortDesc=Les coups critiques appliquent un effet de faiblesse fort
+botania.armorset.will1.shortDesc=Les coups critiques augmentent avec un HP bas
```
Comment (Thog) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: I think you can write something better like "Les coups critiques augmentent quand les HP sont bas".

File: src/main/resources/assets/botania/lang/fr_FR.lang [hunk 3]
```diff
@@ -121,34 +140,35 @@ botania.rank5=&6SS
...
+botania.armorset.will2.shortDesc=Les coups critiques guérit pour une partie des dégâts infligés
```
Comment (Thog) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: It isn't 'guérit' but 'guérissent'... CONJUGATION! Also this sentence doesn't make sense... (that would be better without "pour")

File: src/main/resources/assets/botania/lang/fr_FR.lang [hunk 4]
```diff
@@ -121,34 +140,35 @@ botania.rank5=&6SS
...
-botania.armorset.will5.desc=Critical Hits apply a wither effect (Will of Karil)
-botania.armorset.will0.shortDesc=Critical Hits apply a strong weakness effect
-botania.armorset.will1.shortDesc=Critical Hit damage increases with lower HP
-botania.armorset.will2.shortDesc=Critical Hits heal for a portion of damage dealt
-botania.armorset.will3.shortDesc=Critical Hits apply a strong slowness effect
-botania.armorset.will4.shortDesc=Critical Hits pierce through armor
-botania.armorset.will5.shortDesc=Critical Hits apply a wither effect
+botania.armorset.terrasteel.name=Savoir des Ancients
+botania.armorset.terrasteel.desc0=Coût en Mana diminué de 20% sur les outils à Mana
+botania.armorset.terrasteel.desc1=Régène même si la faim n'est pas remplie
```
Comment (Thog) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: "Régène" ??? This is vulgar!  "Régénère" is more common

Comment (Mazdallier) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: Régène is used by french youtubers so...

File: src/main/resources/assets/botania/lang/fr_FR.lang (hunk 3 above)
Comment (Mazdallier) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: C'est certain...

File: src/main/resources/assets/botania/lang/fr_FR.lang (hunk 2 above)
Comment (Mazdallier) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: T'as un exemple ?

File: src/main/resources/assets/botania/lang/fr_FR.lang (hunk 4 above)
Comment (Thog) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: Yes but that is not a real french word! Imagine that for english translate?

File: src/main/resources/assets/botania/lang/fr_FR.lang (hunk 1 above)
Comment (Mazdallier) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: Ca donne un effet plus role play.

File: src/main/resources/assets/botania/lang/fr_FR.lang (hunk 4 above)
Comment (Mazdallier) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: Moin utilisé par les youtubeur francais

File: src/main/resources/assets/botania/lang/fr_FR.lang (hunk 2 above)
Comment (Thog) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: "Les coups critiques augmentent avec un HP bas"
Sachant que HP = Heal Point = point de vie
"Les coups critiques augmentent avec un point de vie bas", ça ne va pas.

Par contre, "Les coups critiques augmentent quand les points de vies sont bas" ça marche mieux.

This is the last time that I will use french in a PR.

**Comment by Mazdallier** [comment]
More than 2500 lines of translation, of course it Squash.
Verifing every line, adding every missing, making everything at once is impossible.
Trog seems to have much more time than me to make a good translation.
Congratulation...

File: src/main/resources/assets/botania/lang/fr_FR.lang (hunk 1 above)
Comment (Thog) [inline src/main/resources/assets/botania/lang/fr_FR.lang]: Nop, 'Ancient' is not a french word, 'Ancient' is a english word that correspond to 'ancien', I think that Vazkii doesn't want that translation change aspect of the gameplay.

**Comment by Vazkii** [comment]
Closing this since you two can't sort your shit.

**Comment by Mazdallier** [comment]
thanks

**Pull request closed without merging**